client/my-sites/stats/feedback/modal/index.tsx

```diff
@@ -0,0 +1,66 @@
+import { Button, Modal, TextareaControl } from '@wordpress/components';
+import { close } from '@wordpress/icons';
+import { useTranslate } from 'i18n-calypso';
+import React, { useState } from 'react';
+import StatsButton from 'calypso/my-sites/stats/components/stats-button';
+
+import './style.scss';
+
+interface ModalProps {
+	isOpen: boolean;
+	onClose: () => void;
+}
+
+const FeedbackModal: React.FC< ModalProps > = ( { isOpen, onClose } ) => {
```

Comment:
This is another convention (typing the const/function) that I'm not really convinced has much utility. Is there a reason we should adopt this?

Reply:
Several conventions for creating a component with TypeScript will change over time, so I am trying to learn as much about the pros and cons of each method as possible. However, the way of defining a component should not matter much since it would be isolated.

By the way, I just used GitHub Copilot to help me create this template for a new component. 😄